rls/rls-cargo-build-integration-notes.md

```diff
@@ -0,0 +1,58 @@
+# RLS <> Cargo <> External build integration
+
+The problem that is being solved is being able to support build systems other than Cargo by the RLS to get diagnostics and project indexing.
```

Comment:
Could you expand on this - what are the sub-problems? What are milestones or alternatives (without getting into the implementation details.